server/views/pages/adjustments/remand/edit.njk

```diff
@@ -0,0 +1,47 @@
+{% extends "../../../partials/layout.njk" %}
+{% from "./partials/remandSummary.njk" import remandSummary %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set pageTitle = applicationName + " - View remand details" %}
+{% set mainClasses = "app-container govuk-body" %}
+
+{% block beforeContent %}
+  {{ super() }}
+  <nav>
+    {{ govukBackLink({
+      text: "Back",
+      href: "/" + model.prisonerDetail.offenderNo + "/remand/view"
+    }) }}
+  </nav>
+{% endblock %}
+
+{% block content %}
+  <h1 class="govuk-heading-xl"><span class="govuk-caption-xl">Adjust release dates</span>
+    Edit remand details
+  </h1>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+      {{ remandSummary(model.adjustment, model.listOffences(), model.prisonerDetail.offenderNo, true) }}
```

Comment:
Only a suggestion. I like how GDS have done there components where you pass in an object rather than a list of params.

That would make it a bit clearer what the "true" is for in this case:

``` 
{{ remandSummary({
    adjustments: model.adjustment,
    offences:  model.listOffences(),
    offenderNo:  model.prisonerDetail.offenderNo, 
    isEdit: true
}) }}
```

Reply:
yes that looks clearer - will change